pulpcore/app/serializers/fields.py

```diff
@@ -10,6 +10,12 @@
 from pulpcore.app.serializers import DetailRelatedField, RelatedField
 
 
+def relative_path_validator(relative_path):
+    if os.path.isabs(relative_path):
+        raise serializers.ValidationError(_("Relative path can't start with '/'. "
```

Comment:
nitpick - space after `.`.

Reply:
The string continues on the next line. I think, the space is intentional.

```
>>> a = ("asdf"
...     "ghjk")
>>> a
'asdfghjk'
```